### PR TITLE
Added Student Projects Section (Issue #176)

### PR DESCRIPTION
**Pull request by sh4dowb**
https://github.com/fossasia/gci16.fossasia.org/issues/176
Demo: https://hesapadim.github.io/gci16.fossasia.org/

### CODE REVIEW

**Comment by m1guelpf** [comment]
@hesapadim Please:
- Fix the highligth in navbar
- Squash all your commits
- Add an usefull title to the commit

**Comment by sh4dowb** [comment]
@m1guelpf Fixed & added navbar but how can I update commits? I'm new to GitHub.

**Comment by oanarosca** [comment]
@hesapadim Your new commit has been automatically added here. Please see why the travis build failed (maybe you used tabs instead of spaces, left some extra spaces or no line break at EOF), commit your changes and squash them into one commit. If you need help, check out contributors.md or join the slack channel. Good luck!

**Comment by niccokunzmann** [comment]
@hesapadim Have a look into the contributing.md file. There is information about how to squast and force push. Also, you can have a look into the README.md file. There is written, where you can get even more support.

File: index.html [hunk 1]
```diff
@@ -173,6 +174,40 @@
                     {% endfor %}
                 </div>
             </section>
+<<<<<<< HEAD
```
Comment (m1guelpf) [inline index.html:177]: Remove the git tag

File: index.html [hunk 2]
```diff
@@ -173,6 +174,40 @@
...
+                                    </a>
+                                </div>
+                                <p>{{project.name}} by {{project.owner}}</p>
+                            </div>
+                        </div>
+                    </div>
+                    {% endfor %}
+                </div>
+            </section>
+=======
```
Comment (m1guelpf) [inline index.html:209]: Remove the git tag

File: index.html [hunk 3]
```diff
@@ -173,6 +174,40 @@
...
+>>>>>>> parent of 4d27f6b... test
```
Comment (m1guelpf) [inline index.html:210]: Remove the git tag

**Review by m1guelpf — CHANGES_REQUESTED** [review]
Please make the changes addressed

**Comment by niccokunzmann** [comment]
I am merging this, so we can have he projects section now and others can build on it, not later and maybe never because we force some rules. The merge lines are not visible on the website.

**Comment by niccokunzmann** [comment]
@hesapadim Please address the changes. I create an issue.

**Pull request merged**